packages/carousel/src/react/index.js

```diff
@@ -17,13 +17,19 @@ import useResizeObserver from './use-resize-observer.js'
 import useSwipe from './use-swipe.js'
 import useUniqueId from './use-unique-id.js'
 
+const TRANSITION_DURATION_MS = 500
+
 const styles = {
-  carousel: ({ ready }) =>
+  carousel: (props, { ready }) =>
```

Comment:
why add props here?  (not used)

Reply:
props as the first arg(or second if there is theme support 😬 ) seems to be the loose convention we're following so i thought i'd add the props as the first arg